auth/server/src/main/java/com/seagates3/controller/AccountController.java

```diff
@@ -171,6 +190,22 @@ public ServerResponse create() {
           LOGGER.error("Create account delay failing - ", e);
           Thread.currentThread().interrupt();
         }
+        // Handle multi-threaded create API calls by deleting extra accounts,
+        // if account limit creation exceeded due to multiple thread/multiple
+        // node create() API calls.
+        try {
+          accountCount = accountDao.getTotalCountOfAccounts();
```

Comment:
it contains internal accounts also?

Reply:
yes. this will return total ldap entry count which contains all the accounts created including internal accounts.